### PR TITLE
Fixed resolutions with node version 16+

### PR DESCRIPTION
**Pull request by satishnvrn**
We are getting errors like: `<ComponentX> is not a valid JSX element` while running locally with node version 16+ . This PR solves this issue for all such components.

### CODE REVIEW

File: src/components/TrailBlazerModal.tsx [hunk 1]
```diff
@@ -8,6 +8,9 @@ import { VerticalTimeline, VerticalTimelineElement } from "react-vertical-timeli
 import { Loader } from "components";
 import { IFrameModal } from "./iFrameModal";
 import { TwModal } from "./Modal/TwModal";
+import ReactModal from "react-modal";
+
+const ModelR18 = Modal as ComponentType<ReactModal['props']>;
```
Comment (damienen) [inline src/components/TrailBlazerModal.tsx]: I will have to come back with the implications of this change once I can thoroughly test

File: yarn.lock [hunk 2]
```diff
@@ -0,0 +1,14242 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (damienen) [inline yarn.lock:1]: This should be added to .gitignore and deleted from the repo itself as we are using npm in prod

File: src/components/TrailBlazerModal.tsx (hunk 1 above)
Comment (satishnvrn) [reply src/components/TrailBlazerModal.tsx]: This is not required. I deleted this.

File: package.json [hunk 3]
```diff
@@ -102,6 +104,8 @@
     "typescript": "4.1.2"
   },
   "resolutions": {
-    "**/*/nth-check": "2.0.1"
-  }
+    "@types/react": "18.0.24",
+    "@types/react-dom": "18.0.8"
```
[the remark this answers is not in the document]
Comment (satishnvrn) [reply package.json:108]: I got some errors related to types of ReactElement. That's why I added these.

**Pull request closed without merging**